dietpi/dietpi-services

```diff
@@ -194,9 +192,14 @@ Available services:
 			# - Misc
 			'docker'
 			'mycroft'
-			'fahclient'
 			'cron'
 
+			# - Distributed Projects
+			'fahclient'
+			'ipfs'
+			'yacy'
+			'tor'
```

Comment:
Since the Tor service can also be the Tor HotSpot, which needs to be started before hostapd, please keep it in the network section.